apps/sandbox/package.json

```diff
@@ -4,14 +4,14 @@
   "main": "expo-router/entry",
   "scripts": {
     "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios"
+    "android": "expo run:android",
+    "ios": "expo run:ios"
```

Comment:
```suggestion
    "android": "expo start --android",
    "ios": "expo start --ios"
```

Reply:
my bad, this shouldn't have been committed, let me open a PR reverting these changes